site/en/docs/privacy-sandbox/user-agent/snippets/index.md

```diff
@@ -0,0 +1,192 @@
+(Linux; Android <span style="background: #ef9a9a">12; Pixel 5</span>)
+AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.<span  style="background:
+#ef9a9a">0.4638.16</span> Mobile Safari/537.36</span> {% endCompare %}
+
+{% Compare 'better', 'new' %} <span style="font-family: monospace">Mozilla/5.0
+(Linux; Android <span style="background: #a5d6a7">10; K</span>)
+AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.<span style="background:
+#a5d6a7">0.0.0</span> Mobile Safari/537.36</span> {% endCompare %}
+
+## How to override the user-agent in your code
+
+### Local testing methods
+
+- **Enable the `chrome://flags/#reduce-user-agent` flag**
+  - This will enable the new behavior and use the reduced format for the
+    user-agent string for all sites.
+- **Configure an emulated device in DevTools with the right user-agent string
+  and client hints**
+  - In DevTools under ⚙️ Settings → Devices → "Add custom device…" you can
+    configure an emulated device with any combination of user-agent string and
+    user-agent client hints values you need. Use the 📱 "Toggle device toolbar"
+    button to select an emulated device.
+- **Launch Chrome with the `--user-agent="Custom string here"`** 
+  - Use the command line flag to start Chrome with a custom user-agent string.
+
+{% Aside %}
+
+You can also [enroll your own sites in the origin
+trial](https://developer.chrome.com/blog/user-agent-reduction-origin-trial/) to
+enable Chrome browsers visiting your site to send the reduced version of the
+user-agent.
+
+{% endAside %}
+
+As the format for the reduced user-agent string is available, this means you can
+transform and test the new string against your own code—either by overriding and
+replacing it, or by generating the new version and testing side-by-side.
+
+### Regular expression
+
+A regular expression that will **only** match safely transformable Chrome
+user-agents and pulls out the necessary variable values to slot into a template
+for the new format.
+
+All on one line for easy copying. It is long, but that's to prioritise only
+matching against Chrome and doing with relatively cheap checks.
+
+```text
+/^Mozilla\/5\.0 \(((?<platform>Lin|Win|Mac|X11; C|X11; L)+[^\)]+)\) AppleWebKit\/537.36 \(KHTML, like Gecko\) Chrome\/(?<major>\d+)[\d\.]+(?<mobile>[ Mobile]*) Safari\/537\.36$/
+
+```
+
+The expression captures the following values:
+
+*   `platform`: captures the platform or operating system
+*   `major`: captures the browser major version
+*   `mobile`: captures the mobile device indicator
+
+The captured values should be inserted into this template string for the reduced
+format.
+
+```text
+Mozilla/5.0 (${unifiedPlatform[matched.platform]}) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/${matched.major}.0.0.0${matched.mobile} Safari/537.36
+```
+
+The `unifiedPlatform` value comes from taking the short `platform` match and
+replacing it with the appropriate reduced value.
+
+*   `Lin`: `Linux; Android 10; K`
```

Comment:
How about a table here? Not necessary, it's just that no vertical alignment between list items makes it look off. But again, not very important in this case.

Reply:
Unsure about our table formatting… I can't decide if this is an improvement.
![Screenshot 2021-11-26 13 37 01](https://user-images.githubusercontent.com/108052/143589222-24326c3f-d9bf-4818-ae65-04b84e211b4b.png)

Reply:
Awful, don't do it.